docs/api/ShallowWrapper/invoke.md

```diff
@@ -0,0 +1,51 @@
+# `.invoke(event[, ...args]) => Self`
+
+Invoke event handlers
+
+
+#### Arguments
+
+1. `event` (`String`): The event name to be invoked
+2. `...args` (`Any` [optional]): Arguments that will be passed to the event handler
+
+
+
+#### Returns
+
+`ShallowWrapper`: Returns itself.
+
+
+
+#### Example
+
+```jsx
+class Foo extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { count: 0 };
+  }
+  render() {
+    const { count } = this.state;
+    return (
+      <div>
+        <div className={`clicks-${count}`}>
```

Comment:
not actually a blocker ofc, but i'd prefer to use a data attribute here, so that the example follows best practices :-)